### DISCUSSION TITLE
POSTMAN_API_KEY defined as secrets

### DISCUSSION THREAD

**Discussion by cw112233** [apideck-libraries/portman#553] [Q&A]
Hi, I'm facing a challenge to leverage my own CI to execute portman and sync the generated collection to my postman collection. My CI could leverage some secrets file i defined, I know `postmanUid` can be added in CLI as `--postmanUid`, so I can simply replace it as `--postmanUid $UID`, but I'm not able to do that through POSTMAN_API_KEY, since it's not supported to pass in CLI. Any suggestion will be appreciated, thanks!

**Comment by thim81** [comment]
A workaround could be to have the CI dump the ENV variables in a .ENV file

Example, create a CI task

```bash
printenv >> .env
```

**Reply by thim81** [reply]
Or if you want to capture a specific environment variable

```bash
#!/bin/bash

# Specify the name of the environment variable
variable_name="MY_VARIABLE"

# Use env to get the value of the environment variable
variable_value=$(env | grep "$variable_name" | cut -d= -f2)

# Specify the name of the file to write to
output_file=".env"

# Check if the variable is set
if [ -n "$variable_value" ]; then
    echo "$variable_name=$variable_value" > "$output_file"
    echo "Environment variable written to $output_file."
else
    echo "The environment variable $variable_name is not set."
fi

```
